optimum/exporters/neuron/model_configs.py

```diff
@@ -356,12 +357,6 @@ def outputs(self) -> List[str]:
     def generate_dummy_inputs(self, return_tuple: bool = False, **kwargs):
         # For neuron, we use static shape for compiling the unet. Unlike `optimum`, we use the given `height` and `width` instead of the `sample_size`.
```

Comment:
Could you remove these two lines of warning as well? Thx!

Reply:
Yes. removed